core/shared/src/main/scala/fs2/Chunk.scala

```diff
@@ -596,18 +596,24 @@ object Chunk
       case ix: GIndexedSeq[O] => indexedSeq(ix)
       case _ =>
         if (i.isEmpty) empty
```

Comment:
emptiness gets checked in `iterableOnce`, can't we remove this check from here and `L626`?

Reply:
Now the implementation has changed, this comment is outdated